scripts/common/preflights.sh

```diff
@@ -657,3 +658,18 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# bail_when_requires_minio_with_openebs will bail if OpenEBS and Minio not when it is required
+function bail_when_requires_minio_with_openebs() {
+    if [ -z "$MINIO_VERSION" ] && [ -n "$OPENEBS_VERSION" ]; then
```

Comment:
i think you will have to check if rook is included here as well. if it is then there will be an object store.